jupyter/gojupyterscaffold/gojupyterscaffold.go

```diff
@@ -15,12 +15,18 @@ import (
 	"io/ioutil"
 	"os"
 	"os/signal"
+	"os/user"
+	"path"
+	"path/filepath"
+	"sort"
 	"syscall"
 
 	"github.com/golang/glog"
 	zmq "github.com/pebbe/zmq4"
 )
 
+const jupyterStartupDirectoryPath = ".jupyter/startup/"
```

Comment:
How did you decide this path?
Do you have any reference?
I tried to find an official document about the start-up scripts in Jupyter notebook. But I can not find it. Instead, I found some documents mention `.ipython/profile_default/startup`.